### PR TITLE
577 patroni postgres role

### PR DESCRIPTION
**Pull request by kayiwa**
This installs a patroni HA cluster

### CODE REVIEW

**Comment by jrgriffiniii** [comment]
This is passing for me locally, https://github.com/pulibrary/princeton_ansible/pull/870/checks?check_run_id=234499899 was canceled, so perhaps that is why the build status indicates that it is failing?

File: roles/pulibrary.patroni/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+patroni_dcs: consul
+patroni_dcs_exists: false
+
+patroni_exec_start_pre: "/bin/mkdir -m 2750 -p /var/run/postgresql/{{ patroni_postgresql_version }}-main.pg_stat_tmp"
+
+patroni_replication_username: replicator
+patroni_replication_password: repuserpasswd
+patroni_superuser_username: postgres
+patroni_superuser_password: supersecretpostgrespasswd
```
Comment (jrgriffiniii) [inline roles/pulibrary.patroni/defaults/main.yml]: Should these values be imported from Vault values or are they overridden later?

Comment (jrgriffiniii) [reply roles/pulibrary.patroni/defaults/main.yml]: kayiwa directed me to https://github.com/pulibrary/princeton_ansible/blob/577_patroni_postgres_role/group_vars/patroni/staging.yml, where it is indeed set.

Comment (kayiwa) [reply roles/pulibrary.patroni/defaults/main.yml]: overridden by `group_vars/patroni/` variables (I made an issue) however. https://github.com/pulibrary/princeton_ansible/issues/872

**Pull request merged**